crates/quickjs-wasm-rs/src/serialize/de.rs

```diff
@@ -245,16 +245,14 @@ mod tests {
     fn test_null() {
         let context = Context::default();
         let val = context.null_value().unwrap();
-        let actual = deserialize_value::<()>(val);
-        assert_eq!((), actual);
+        deserialize_value::<()>(val);
```

Comment:
This will always return `()` because of the generic type being passed so Clippy complains about the let binding and the assert check. I think as long as this runs without panicking, it should be fine IMO.